src/LogExpert/Config/ColorMode.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LogExpert.Config
+{
+    public static class ColorMode
+    {
+        // Bright Theme
+        // https://paletton.com/#uid=15-0u0k00sH00kJ0pq+00RL00RL
+        private static System.Drawing.Color BrightBookmarkDefaultSystemColor = System.Drawing.SystemColors.Control; // Important: only supports SystemColors
```

Comment:
add the Using System.Drawing and so that you can remove it from the static variables, makes them more readable

Reply:
I am not sure I understand how to do that, I only know "Using" like Using (something) { }

Reply:
na I mean the "Usings" top of the file, sorry for the miscommunication 😄 

Using System.Drawing;

then you don't need it with all the variables and can just use "Color x = SystemColors.X"

Reply:
Aaah I see sorry, done!